resources/js/components/Committee/CommitteeSessions.vue

```diff
@@ -97,7 +97,11 @@ export default {
             {{ `${dropValue.label} (${sortedList.length} Sitzungen)` }}
         </h2>
         <div class="ris-calendar">
-            <CalendarCard class="ris-calendar__card-list" v-for="(meeting, index) in sortedList " :key="index" :meeting-sorted-day-list="meeting" />
+            <CalendarCard
+                class="ris-calendar__card-list"
+                v-for="(meeting, index) in sortedList "
+                :key="index"
```

Comment:
is it possible to use something that is unique for real instead of `index`? afaik using `index` could cause unexpected behaviors during rendering

Reply:
is fixed